tests/html/Element/removeAttributeNS.html

```diff
@@ -102,6 +102,20 @@
     assert.equal(element.attrCallbackArgs.length, 2);
     assert.deepEqual(element.attrCallbackArgs[1], ['attr', 'abc', NON_EXISTANT_VALUE, 'ns']);
   });
+
+  test('Removing an observed attribute with a different capitalization does not trigger a callback.', function() {
+    const element = document.createElement(localName2);
+
+    assert.equal(element.attrCallbackArgs.length, 0);
+
+    element.setAttributeNS('ns', 'attr', 'abc');
```

Comment:
I don't think that the lowercasing behavior applies to `{set,remove}AttributeNS`. This passes in a native implementation?

Reply:
You are correct. This test therefore ensures that when a different capitalization is used (e.g. `Attr`), the callback is NOT invoked.

Reply:
> the callback is NOT invoked

Ah ok, thanks for the clarification.